subt/jetson_artf_node.py

```diff
@@ -0,0 +1,66 @@
+"""
+  Jetson artf detector wrapper
+"""
+import os.path
+from io import StringIO
+
+import cv2
+import numpy as np
+from subt.tf_detector import CvDetector
+
+from osgar.node import Node
+from subt.artf_node import result2report
+from osgar.bus import BusShutdownException
+
+
+class ArtifactDetectorJetson(Node):
```

Comment:
what is the relation/difference to existing detector nodes?

Reply:
I would like to keep these files separately. It is used only one detector with different thresholds and there is different approach for artefact localization.